docs/http.md

```diff
@@ -0,0 +1,61 @@
+# HTTP API
+
+When a Cog Docker image is run, it serves an HTTP API for making predictions.
+
+First, build your model:
+
+    cog build -t my-model
+
+Then, start the Docker container:
+
+    docker run -d -p 5000:5000 --init my-model
+
+    # If your model uses a GPU:
+    docker run -d -p 5000:5000 --init --gpus all my-model
+
+    # if you're on an M1 Mac:
+    docker run -d -p 5000:5000 --init --platform=linux/amd64 my-model
```

Comment:
Have you got this working with [M1's GPU yet](https://nod.ai/pytorch-m1-max-gpu/)?

Reply:
Unfortunately I think it's very very hard to navigate through xhyve etc.

Reply:
I'm hoping Apple invests in it.... would make the Ultra and whatever Mac Pro is to come more appealing

Reply:
Yeah 100%.

It's starting to work in some form, but it only works if you run PyTorch and Tensorflow on the bare metal (and deal with dependency hell).